packages/task/src/node/task-server.slow-spec.ts

```diff
@@ -237,12 +238,16 @@ describe('Task server / back-end', function () {
         });
 
         const signal = await p;
-        expect(signal).equals('SIGTERM');
+        // expect(signal).equals('SIGTERM');
+        expect(signal).equals('SIGHUP'); // ?
```

Comment:
Ah, apparently it happened with other test.

My guess is that because when using node-pty, the processes are slaves to some master pty.
So when the slave exits, it would translate to a SIGHUP?

Reply:
I would venture and say that it's because it's the signal sent by defaut by node-pty when killing a process.

https://github.com/Microsoft/node-pty/blob/master/src/unixTerminal.ts#L226